src/components/todoitems/TodoItems.js

```diff
@@ -4,28 +4,28 @@ import { FaTrash } from 'react-icons/fa';
 
 function TodoItems({ onCheck, onClick, data }) {
 
-    const labelRef = useRef()
+    const labelRef = useRef();
 
     useEffect(() => {
-        const label = labelRef.current
-        label.className = data.completed ? "todoTextComplete" : "todoText"
-    }, [data.completed])
+        const label = labelRef.current;
+        label.className = data.completed ? "todoTextComplete" : "todoText";
```

Comment:
u don't need a ref to set className, u can directly do it using template string

Reply:
done